website/src/components/demos/pagination.demo.tsx

```diff
@@ -1,38 +1,7 @@
-import { ChevronLeftIcon, ChevronRightIcon } from 'lucide-react'
-import { Button } from '~/components/ui/button'
-import { IconButton } from '~/components/ui/icon-button'
-import type { PaginationProps } from '~/components/ui/pagination'
-import * as Pagination from '~/components/ui/pagination'
+import { Pagination, type PaginationProps } from '~/components/ui/pagination'
```

Comment:
I found type props that the previous few examples import are from the `ark/react` library instead of `~/components/ui/<component>`. Should I go to fix them?